modules/seqexec/model/shared/src/main/scala/seqexec/model/ObservationProgress.scala

```diff
@@ -23,6 +25,12 @@ object Progress {
       case (a: NSObservationProgress, b: NSObservationProgress) => a === b
       case _                                                    => false
     }
+
+  implicit val obsProgressP: Prism[Progress, ObservationProgress] =
```

Comment:
This seems a bit strange, having an implicit `Prism` but it seems to work fine

Reply:
Yup, it's used at https://github.com/gemini-hlsw/ocs3/blob/0f625eb4d91c002956171ae280eb8761a2ff1086/modules/seqexec/web/client/src/main/scala/seqexec/web/client/circuit/SeqexecCircuit.scala#L142 so that we can request a subclass to the lens through type parameters.